### PR TITLE
Remove public chats

### LINKED ISSUE #8504: Remove public chats
We are going want to remove the public unmoderated chat functionality before the MVP release (as this will be replaced with Status Communities).  Opening this issue so that this is tracked somewhere.  There isn't any reason to keep this functionality in Status Desktop at this point, so it can be removed in any of the upcoming releases.

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #8504

Removes the ability to create public chats, see public chats and even leaves previously active public chats

Also, removes e2e tests that use public chats. I added issues to re-add them in the context of community channels.

### CODE REVIEW

**Review by mprakhov — APPROVED** [review]
That's a big one, lgtm 👍

**Review by borismelnik — APPROVED** [review]
LGTM

**Review by richard-ramos — APPROVED** [review]

File: ui/imports/shared/views/chat/MessageContextMenuView.qml [hunk 1]
```diff
@@ -29,7 +29,7 @@ StatusMenu {
     property string selectedUserDisplayName: ""
     property string selectedUserIcon: ""
 
-    property int chatType: Constants.chatType.publicChat
+    property int chatType: Constants.chatType.unknown
```
Comment (MishkaRogachev) [inline ui/imports/shared/views/chat/MessageContextMenuView.qml:32]: Isn't it should be 'oneToOne'? `unknown` is not handled in that [switch](https://github.com/status-im/status-desktop/pull/9253/files#diff-54c7d5c23951abe7c4bd4df905b5fc2a58bbdd03ba320183e128eb8d7d17830aR413)

**Review by MishkaRogachev — APPROVED** [review]
Looks good for me! No public chats - less problems)

**Comment by jrainville** [comment]
Tests are actually green on CI. They reported as failed, but the retries actually worked

**Comment by anastasiyaig** [comment]
this is tested and looks great

1. communities are still working (adding, removing channels, etc)
2. upgrading to this pr having older version with variety of public chats is working fine (no public chats appear, messaging to public chats when they are already removed does not force them to appear)

Thanks @jrainville !

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**